vocabularies/Org.OData.Capabilities.V1.xml

```diff
@@ -995,6 +995,72 @@ The absence of `RestrictedProperties` denotes all properties are accessible usin
         <Annotation Term="Core.Description" String="Stream property supports update of its media edit URL and/or media read URL" />
       </Term>
 
+      <Term Name="DefaultCapabilities" Type="Capabilities.DefaultCapabilitiesType" Nullable="false" AppliesTo="EntityContainer">

```

Comment:
```suggestion
      <Term Name="Capabilities" Type="Capabilities.DefaultCapabilitiesType" Nullable="false" AppliesTo="EntityContainer Collection">
  <Annotation Term="Core.AppliesViaContainer"/>
```
This term could also apply to a collection-valued (complex) _structural_ property, using external targeting at `Container/EntitySet/CollectionValuedProperty`. (We cannot use `NavigationRestrictions` in such a case, because there is no navigation proeprty involved.)

Reply:
Please don't name a term identical to the alias of the vocabulary it is in, I prefer "DefaultCapabilities".

Reply:
TC discussion: No need to specify capabilities for individual `Container/EntitySet/CollectionValuedProperty`s.